legend-pure-core/legend-pure-m3-core/src/main/java/org/finos/legend/pure/m3/exception/PureExecutionException.java

```diff
@@ -137,7 +137,14 @@ public <T extends Appendable> T printPureStackTrace(T appendable, String indent,
                 CoreInstance func = x.getValueForMetaPropertyToOne(M3Properties.func);
                 if (func != null)
                 {
-                    FunctionDescriptor.writeFunctionDescriptor(appendable, func, false, processorSupport);
+                    try
+                    {
+                        FunctionDescriptor.writeFunctionDescriptor(appendable, func, false, processorSupport);
+                    }
+                    catch (IllegalArgumentException e)
+                    {
+                        safeAppendable.append("NULL / TODO");
+                    }
```

Comment:
We should be able to identify and handle properties, rather than just writing "NULL / TODO".